dashboard-app/src/dashboard/ByActivity/index.tsx

```diff
@@ -132,25 +102,30 @@ const ByActivity: FunctionComponent<RouteComponentProps> = (props) => {
           />
         </Col>
       </Row>
-      {loadingLogs || loadingAct || loadingVols
-      ? (<FullScreenBeatLoader color={ColoursEnum.purple}/>)
-      : (<Row center="xs">
-        <Col xs={9}>
-        <Errors errors={errors}/>
-          {
-            tableData && (
-              <DataTable
-                {...tableData}
-                title={TABLE_TITLE}
-                sortBy={tableData.headers[sortBy]}
-                initialOrder="desc"
-                onChangeSortBy={onChangeSortBy}
-                showTotals
-              />
-            )
-          }
-        </Col>
-      </Row>)
+      {
+        loading
```

Comment:
we should remove the loading spinner on subsequent requests, but we can do this in another PR.